applications/data_analysis/climatology.ipynb

```diff
@@ -0,0 +1,3551 @@
+{
```

Comment:
<p>(from convo)</p>

<pre class="ql-syntax" spellcheck="false">Line #9.    # Calculate the standard deviation from monthy mean data
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>